src/main/java/de/cuuky/varo/game/start/ProtectionTime.java

```diff
@@ -20,9 +21,14 @@ public ProtectionTime(int timer) {
 	}
 
 	private void startGeneralTimer(int timer) {
-		this.scheduler = new BukkitRunnable() {
+		this.protectionTimer = timer;
 
-			private int protectionTimer = timer;
+		if (this.protectionTimer == 0) {
+			Main.getVaroGame().setProtection(null);
+			return;
+		}

```

Comment:
Why was this added?

Reply:
When I tried to use the remaining protection time in the Actionbar, there were a lot of error messages after the protection time was over, this seemed to fix it.

Reply:
That doesn't really make sense. `timer` would have to be 0 for this to do anything. But timer can't be 0 because that is already checked in `VaroGame`. So unless I am missing something that if statement does absolutely nothing

Reply:
> That doesn't really make sense. `timer` would have to be 0 for this to do anything. But timer can't be 0 because that is already checked in `VaroGame`. So unless I am missing something that if statement does absolutely nothing

This just fixed the issue where after the protection time it started spamming an error in the console. I would keep this if statement in just for safety.

Reply:
But it doesn't do anything unless you call the constructor from another plugin

Reply:
> But it shouldn't do anything unless you call the constructor from another plugin
> 
> > This just fixed the issue where after the protection time it started spamming an error in the console. I would keep this if statement in just for safety.
> 
> What was the error?

Can't reproduce the error right now but in my plugin, I'm using `Main.getVaroGame().getProtection() == null` for an if statement

Reply:
```suggestion
		if (this.protectionTimer == 0) {
			throw new IllegalArgumentException();
			return;
		}
```
This would be ok

But you probably had a different error somewhere else in your plugin

Reply:
> This would be ok
> 
> But you probably had a different error somewhere else in your plugin

I would just keep `Main.getVaroGame().setProtection(null);` in there. Before I had `Main.getVaroGame().setProtection(null);` in the Varo Plugin, it made the errors.